File: PlayMinds-Back/src/main/java/com/coderhouse/Entitys/Comment.java
package com.coderhouse.Entitys;
import jakarta.persistence.*;
import lombok.Data;

import java.io.Serializable;
import java.sql.Date;

import org.springframework.data.annotation.Id;

@Data
@Entity
@Table(name = "comments")
public class Comment implements Serializable {
    @Id
    private String game_id_game;
    @Id
    private String user_email;
    private String comment;
    private Date date_comment;
}

File: play-minds/src/models/Entitys/Comment.ts
export class Comment {
  id_comment?: number;
  game_id_game: string;
  user_email: string;
  comment: string;
  date_comment: Date;
  constructor(
    id_comment: number,
    game_id_game: string,
    user_email: string,
    comment: string,
    date_comment: Date
  ) {
    this.id_comment = id_comment;
    this.game_id_game = game_id_game;
    this.user_email = user_email;
    this.comment = comment;
    this. date_comment =  date_comment;
  }
}